scripts/plugins/critical-log-review/config.yaml

```diff
@@ -0,0 +1,90 @@
+# Critical Log Review Configuration
```

Comment:
please put the config file in the `prow-jobs/pingcap/tidb` folder

split the config and tool codes.

Reply:
This config is shared by all repositories and should not be scattered in multiple places.

Reply:
updated